### PR TITLE
adding storybook and stuff

### PR DESCRIPTION
**Pull request by joeycozza**

### CODE REVIEW

File: packages/react-scripts/scripts/utils/frontierInit.js [hunk 1]
```diff
@@ -61,148 +61,155 @@ function installFrontierDependencies(appPath, appName, answers, ownPath) {
...
   devDepsToInstall.push(
     ...[
+      '@storybook/addon-actions@5',
+      '@storybook/addon-console@1',
+      '@storybook/addon-info@5',
+      '@storybook/addon-knobs@5',
+      '@storybook/addon-viewport@5',
+      '@storybook/addons@5',
+      '@storybook/react@5',
+      'storybook-readme@5',
```
Comment (redbugz) [inline packages/react-scripts/scripts/utils/frontierInit.js]: I wonder if we should have a wrapper project that brings in the common bits?

File: packages/react-scripts/template/.storybook/webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+const custom = require('@fs/react-scripts/config/zion-webpack.config.js')
+
+module.exports = async ({ config, mode }) => {
+  // There are issues with storybook and lerna monorepos. Following the issue here,
```
Comment (redbugz) [inline packages/react-scripts/template/.storybook/webpack.config.js]: does this still apply to CRA?

**Comment by redbugz** [comment]
Tests failing with: ```/home/travis/tmp/my-test-app/src/components/user/UserInfo.stories.js
  1:8  error  'React' is defined but never used  no-unused-vars
✖ 1 problem (1 error, 0 warnings)```

**Comment by joeycozza** [comment]
Fixed the test.

**Pull request merged**